sentry/values.yaml

```diff
@@ -1430,13 +1420,20 @@ kafka:
       - name: profiles
       - name: ingest-occurrences
       - name: snuba-spans
+  listeners:
+    client:
+      protocol: "PLAINTEXT"
+    controller:
+      protocol: "PLAINTEXT"
+    interBroker:
+      protocol: "PLAINTEXT"
+    external:
+      protocol: "PLAINTEXT"
   zookeeper:
-    enabled: true
-  kraft:
     enabled: false
-  service:
-    ports:
-      client: 9092
+    replicaCount: 3
```

Comment:
Why do you change that value here?

Reply:
Mistake with https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml#L1337